emojivoto/README.md

```diff
@@ -9,6 +9,8 @@ The application is composed of the following 3 services:
 * [emojivoto-emoji-svc](emojivoto-emoji-svc/): gRPC API for finding and listing emoji
 * [emojivoto-voting-svc](emojivoto-voting-svc/): gRPC API for voting and leaderboard
 
+![alt text](assets/emojivoto-topology.png "Emojivoto Topology")
```

Comment:
Maybe instead of "alt text" this should say "topology" or something similar?

Reply:
ahhh whoops nice catch!